relay/kusama/src/lib.rs

```diff
@@ -678,6 +687,17 @@ pub mod dynamic_params {
 		#[codec(index = 4)]
 		pub static UseAuctionSlots: bool = true;
 	}
+
+	/// Parameters used by `pallet-treasury` to handle the burn process.
+	#[dynamic_pallet_params]
+	#[codec(index = 1)]
+	pub mod treasury {
+		#[codec(index = 0)]
+		pub static BurnPortion: Permill = Permill::from_percent(0);
+
+		#[codec(index = 1)]
+		pub static BurnDestination: BurnDestinationAccount = Default::default();
```

Comment:
```suggestion
		pub static BurnDestination: Option<AccountId> = None;
```

Any reason why we need a new struct?

Reply:
Yes. Mentioning `AccountId` explicitly, as well as `Option<AccountId>` causes `#[dynamic_pallet_params]` macro parsing to fail.

Still trying to figure out the reasons behind, so I'll raise an issue about it today. Meanwhile, the new struct is a good, codec-compatible workaround to that issue.